tests/e2e/specs/onboarding.js

```diff
@@ -0,0 +1,25 @@
+import { browser, expect } from '@wdio/globals';
+
```

Comment:
how about a different folder structure?
```
tests/e2e/onboarding.spec.js
tests/unit/adblocker.spec.js
```

Reply:
There are some configuration files, which will share the same folder with specs then. I am not sure. I would keep the `specs` subpath.

Reply:
this is also my concern, I would prefer to be able to tell if a file is a test or a util based on its name, hence `.spec.` infix.

Reply:
We can go with the suffix, no problem. I'll try flattening the structure, it might be not that bad.